mvp/resources/README

```diff
@@ -1 +1,71 @@
-Dummy readme
\ No newline at end of file
+IBM Distribution for Galasa version 1.0 (20210618)                                                                                                                
+===============================================================================      
+
+Overview                    
+------------                                                                                                                                     
+                                                                                                              
+Start using Galasa to enable deep integration testing across platforms and technologies, and run 
+repeatable, reliable, agile testing at scale across your enterprise.   
+
+This zip file contains:                                                                                                                             
+    - An Eclipse directory containing the Galasa plug-in                                                                                                
+    - A Maven directory containing dependencies that are required for building Galasa tests                                                             
+    - A Javadoc directory containing Javadoc API documentation for the Galasa Managers                                                                                                                              
+    - An isolated.tar file - an optional Docker image that hosts the Eclipse and Maven directories. Use this file to host Galasa                        
+    on an internal server that can be accessed by other users.                                                                                          
+    - A docs.jar file that enables you to run the Galasa website locally on your machine                                                                
+    - A Notices file                                                                                                                                    
```

Comment:
IBM Distribution for Galasa 1.0 Notices.html

Reply:
updated